cmd/influxd/inspect/build_tsi/build_tsi_test.go

```diff
@@ -356,9 +356,14 @@ func runCommand(t *testing.T, params cmdParams, outs cmdOuts) {
 	}
 
 	if outs.expectCompactSeries {
+		sfile := tsdb.NewSeriesFile(outs.sfilePath)
+		require.NoError(t, sfile.Open())
+		defer sfile.Close()
+
 		// Get size of all partitions before series compaction
-		beforeSize, err := outs.sfile.FileSize()
+		beforeSize, err := sfile.FileSize()
 		require.NoError(t, err)
+		require.NoError(t, sfile.Close())
```

Comment:
Is this close needed since it was deferred a few lines earlier? Also a `FileSize()` is called on it later, but I don't know if that will cause a problem if the file is closed or not

Reply:
It's needed because when `cmd.Execute()` runs below, it tries to delete/move the file(s) pointed-to by `sfile`.

Good call about the `FileSize()`... the `afterSize` is probably always 0 right now since we aren't re-opening it. I'll fix.

Reply:
Done!